src/sugar.jl

```diff
@@ -1162,31 +1162,31 @@ end
 
 
 """
-    seeded_autodiff_thunk(
+    autodiff(
         rmode::ReverseModeSplit,
-        dresult,
-        f,
-        ReturnActivity,
+        f::Annotation,
+        ReturnActivity::Type{<:Annotation},
+        dresult::Seed,
```

Comment:
I guess it's hard to figure out if `dresult` implies `Duplicated` or `Active`? 

We should have an error check that `ReturnActivity` can't be "Const"

Reply:
I added the `Const` check